index.html

```diff
@@ -0,0 +1,132 @@
+  </head>
+  <!----Title End---->
+
+  <!----Body start---->
+  <body>
+    <!-----Header---->
+    <div class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-----Header Ends----->
+
+      <nav>
+        <!------Changed to nav then changed in css------>
+        <ul>
+          <li>
+            <a href="#search-engine-optimization">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </div>
+
+    <!----Main body of website---->
+    <main>
+      <div class="hero"></div>
+      <div class="content">
+        <!------Changed to individual sections----->
+        <section id="search-engine-optimization" class="paragraph-heading">
+          <img
+            src="./assets/images/search-engine-optimization.jpg"
+            alt="A book full of ideas"
+            class="float-left"
+          />
+          <h2>Search Engine Optimization</h2>
+          <p>
+            The dominance of mobile internet use means that users are searching
+            for the right business as they travel, shop, or sit on their couch
+            at home. Search Engine Optimization (SEO) allows you to increase
+            your visibility and find the right customers for your business.
+          </p>
+        </section>
+
+        <section id="online-reputation-management" class="paragraph-heading">
+          <img
+            src="./assets/images/online-reputation-management.jpg"
+            alt="A figure looks at his phone whilst they try to get company reputation up on a laptop"
+            class="float-right"
+          />
+          <h2>Online Reputation Management</h2>
+          <p>
+            The web is full of opinions, and some of these can be negative.
+            Social media allows anyone with an internet connection to say
+            whatever they want about your business. Online Reputation Management
+            gives you the control over what potential customers see when they
+            search for your business.
+          </p>
+        </section>
+
+        <section id="social-media-marketing" class="paragraph-heading">
+          <img
+            src="./assets/images/social-media-marketing.jpg"
+            alt="A group at a table discussing ideas"
+            class="float-left"
+          />
+          <h2>Social Media Marketing</h2>
+          <p>
+            Social media continues to have a sizable influence on buying habits.
+            Social media marketing helps you determine which platforms are
+            suited to your brand, using analytics to find the right markets and
+            increase your lead generation.
+          </p>
+        </section>
+      </div>
+
+      <section class="benefits">
```

Comment:
This could be an `aside` tag instead of a section.

Reply:
I have now changed the section tag to aside